### DISCUSSION TITLE
How to integrate with Alembic?

### DISCUSSION THREAD

**Discussion by flyoozes** [aws/aws-advanced-python-wrapper#651] [Q&A]
Hey! Are there examples how to integrate it with alembic? When I'm trying to pass `AwsWrapperConnection` to alembic `context.configure()` I receive and error, because connection doesn't have dialect...
Is there a way to associate `AwsWrapperConnection`  with alembic context? 
Thank you!

**Comment by karenc-bq** [comment]
Hi @flyoozes, we don't have any examples on integrating the wrapper with Alembic at the moment. We will look into it and update you when we have more examples.
